src/problem.jl

```diff
@@ -171,9 +171,12 @@ function JumpProblem(prob, aggregator::AbstractAggregatorAlgorithm, jumps::Abstr
                      kwargs...)
     JumpProblem(prob, aggregator, JumpSet(jumps...); kwargs...)
 end
-function JumpProblem(prob, jumps::JumpSet; kwargs...)
-    JumpProblem(prob, NullAggregator(), jumps; kwargs...)
+function JumpProblem(prob, jumps::JumpSet; vartojumps_map=nothing, jumptovars_map=nothing, dep_graph=nothing, spatial_system=nothing, hopping_constants=nothing, kwargs...)
```

Comment:
Don't you need versions for all the jump types too? i.e. like
```function JumpProblem(prob, jumps::ConstantRateJump; kwargs...)```
and
```function JumpProblem(prob, jumps::AbstractJump...; kwargs...)```

Reply:
See the dispatches above your new one.

Reply:
These should probably get associated tests too.

Reply:
The dispatches on lines 138-152 above already exist and have the signature you specified. Why they already exist is unclear to me but I did not have to add them.